api/task_create_test.go

```diff
@@ -170,21 +170,29 @@ func TestTaskLifeCycleHandler_CreateTask_InternalError(t *testing.T) {
 			message:    "error initializing new task, mock error",
 			statusCode: http.StatusInternalServerError,
 		},
+		{
+			name:       "happy_path_run_now",
```

Comment:
the test name does not seem to reflect the test case (looks unhappy)

Reply:
it is very unhappy